data/campaigns/Heir_To_The_Throne/units/Dark_Queen.cfg

```diff
@@ -60,9 +64,15 @@
             name=scepter
         [/filter_attack]
         start_time=-200
+        swoosh_start_time=-50
         [frame]
-            image="units/human-queen-staff-[1~4].png:[100*3,75]"
+            image="units/human-queen.png:100,units/human-queen-attack.png:200,units/human-queen.png:1"
         [/frame]
+        [swoosh_frame]
+            image="units/human-queen-swoosh.png:100,misc/blank-hex.png:1"
+	    auto_vflip=no
```

Comment:
Yeah, he has some lines. A portrait for him might be nice, but it's not a high priority.